cpg_infra/billing_aggregator/gcp_cost_report_slack_bot/main.py

```diff
@@ -409,7 +415,7 @@ def post_slack_message(
     is_monday = datetime.now(tz=TIMEZONE).weekday() == 0
 
     # Only post on Mondays or when a project gets flagged
-    if not is_monday and len(flagged_projects) < 1:
+    if not force_run and not is_monday and len(flagged_projects) < 1:
```

Comment:
Just checking that Hail is always flagged here right - so we'll always get a cost update (just a lot smaller)

Reply:
Yup, hail is always a flagged project.